tests/Zend/Session/SessionTest.php

```diff
@@ -85,7 +85,8 @@ function setUp()
      */
     public function tearDown()
     {
-        ini_set('session.save_path', $this->_savePath);
+        session_abort();
```

Comment:
The lines added here stop the session so that we can change the session save_path (php 7.2 emits a warning that this can't be done while the session is active).